src/test/java/com/team4/testingsystem/services/impl/TestsServiceImplTest.java

```diff
@@ -187,4 +187,60 @@ void removeFail() {
 
     }
 
+    @org.junit.jupiter.api.Test
+    void assignCoachSuccess() {
+
+        Mockito.when(usersService.getUserById(1L)).thenReturn(user);
+
+        Mockito.when(testsRepository.assignCoach(user,1L)).thenReturn(1);
+
+        testsService.assignCoach(1L, 1L);
```

Comment:
please, use different values for different proposes

Reply:
I use 1 as good id and 42 as bad in all my tests. I can add something like `final long GOOD_ID = 1L` and `final long BAD_ID = 42L` to avoid magic numbers